dev-lang/mujs/mujs-0_p20161031.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils flag-o-matic toolchain-funcs vcs-snapshot
+
+DESCRIPTION="lightweight Javascript interpreter"
+HOMEPAGE="http://mujs.com/"
+SRC_URI="http://git.ghostscript.com/?p=mujs.git;a=snapshot;h=a0ceaf5050faf419401fe1b83acfa950ec8a8a89;sf=tgz -> ${P}.tar.gz"
+
+LICENSE="AGPL-3"
+SLOT="0"
+KEYWORDS="~alpha amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~ppc-macos ~x64-macos ~x86-macos"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	default
+	eapply "${FILESDIR}"/${PN}-0_p20150202-Makefile.patch
```

Comment:
A random example for a conversion to `PATCHES` array: c3c7b1fd5e5c3f2925ad71c73bd5ae12eb2e5956
It is just the preferred style in EAPI 6 to use `PATCHES` instead of directly calling `eapply`. It behaves the same way as the variable with the same name in `distutils-r1.eclass`:
https://devmanual.gentoo.org/eclass-reference/distutils-r1.eclass/index.html